test/test.js

```diff
@@ -1,9 +1,12 @@
 const chai = require('chai');
 const chaiHttp = require('chai-http');
+const nock = require('nock');
 
 const server = require('../index');
+const github = require('../services/github');
 
 const should = chai.should();
+const scope = nock('https://api.github.com');
```

Comment:
I would choose a more descriptive name which implies that it's a mocked API, like `githubAPIMock`. It could easily happen that we need multiple mocks.